sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -2740,6 +2740,49 @@ public void onTimer(OnTimerContext context) {
     pipeline.run();
   }
 
+  @Ignore
+  @Test
+  @Category({ValidatesRunner.class, UsesStatefulParDo.class})
```

Comment:
<!--new_thread; commit:2a4d086c9e41b1bfcc495a166dac43e68bb82dbe; resolved:0-->
UsesTimersInParDo?

Reply:
Done, and adding JIRA links to the `@Ignore`. I'll make tests actually do pass after rebase. Don't recall why they didn't previously, since this is a noop for them. Probably some past flake.